scripts/wagyu/0.4.2/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: cpp
```

Comment:
Technically this can be `language: generic`, which would boot faster. That is not going to matter, but it is what I prefer.